packages/address-book/address-book/fantom/tokens/tokens.ts

```diff
@@ -904,6 +904,17 @@ const _tokens = {
     description:
       "Terra's native token, LUNA, is used to stabilize the price of the protocol's stablecoins. LUNA holders are also able to submit and vote on governance proposals, giving it the functionality of a governance token.",
   },
+  asUSDC: {
+    name: 'USDC Solana',
+    address: '0xb5E4D17FFD9D0DCE46D290750dad5F9437B5A16B',
+    symbol: 'USDC',
```

Comment:
this should be asUSDC?

Reply:
Yes, it's corrected in the latest commit.